front/components/assistant/conversation/ConversationSidePanelContext.tsx

```diff
@@ -26,8 +26,6 @@ type OpenPanelParams =
 
 interface AgentActionState {
   actionProgress: ActionProgressState;
-  isActing: boolean;
```

Comment:
Why those changes?

Reply:
to fix the spinner ! we actually don't need to pass all that metadata, we can infer it from the agent message in the panel